### PR TITLE
fix: welcome section header tm-25

### PR DESCRIPTION
**Pull request by MappariChan**
fix welcome section header subtitle line height and add overall overview page padding

### CODE REVIEW

**Review by YourTeaChan — APPROVED** [review]

File: apps/frontend/package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
 		"react-redux": "9.1.0",
 		"react-router-dom": "6.21.3",
 		"react-toastify": "10.0.4",
-		"shared": "*"
+		"shared": "*",
+		"vite-plugin-svgr": "4.2.0"
```
Comment (ShadowOfTheSpace) [inline apps/frontend/package.json:30]:
```suggestion
		"shared": "*"
```
We have this package in devDependencies already (Vlad changed it recently [commit])(https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/commit/da822e6292963015a4c93b7cbfac6cce49d72dde)

**Review by ShadowOfTheSpace — CHANGES_REQUESTED** [review]

**Review by GEOFARL — APPROVED** [review]

File: apps/frontend/package.json (hunk 1 above)
Comment (what1s1ove) [reply apps/frontend/package.json:30]: right

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by fshabanov — CHANGES_REQUESTED** [review]
Add the screenshot of a final result

**Review by Sytnikovakatya — APPROVED** [review]

**Comment by MappariChan** [comment]
> Add the screenshot of a final result

image

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**